repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/tracing/rewritetransid/system-model.cfg.xml

```diff
@@ -13,4 +13,5 @@
                       default="true"/>
         </destinations>
     </repose-cluster>
+    <tracing-header rewrite-header="true"/>
```

Comment:
Making it mandatory doesn't fix the problem, because it's a change in a config that we don't replace on upgrade.

Our usage breaks down like this:
97% Want the header on
3% Want rewrite
0% Want it turned off (I haven't heard of anybody not wanting the header, but we give the option, because there is always one weirdo out there)

I don't want to make people have configuration for something that they want all the damn time, it's the opposite of convention over configuration which is where the industry is moving. We are still giving them the option of being explicit if they want, but why specify something that is always going to be the same. It's only if you need to change it's default behavior that you need to specify it and interact with it.

make sense?